.releaserc

```diff
@@ -4,13 +4,19 @@
       "preset": "angular",
       "releaseRules": [
         {"type": "chore", "release": "patch"}
-        
+
       ]
     }],
-    "@semantic-release/npm",
     "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/changelog",
+      {
+        "changelogFile": "CHANGELOG.md"
+      }
+    ],
+    "@semantic-release/npm",
     ["@semantic-release/git", {
-      "assets": ["docs", "package.json"],
+      "assets": ["docs", "package.json", "CHANGELOG.md"],
       "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
     }],
     ["@semantic-release/github", {
```

Comment:
do we have duplicated artefacts upload actions in this repo: https://github.com/appium/io.appium.settings/blob/2e5f82d9b7d149fa816ec58a234c9f1cbce6d100/.github/workflows/publish.js.yml#L35 ?

Reply:
Oh, maybe I forgot removing that one. I'll remove it